### DISCUSSION TITLE
Allow a firebase authentication user to connect OpenSea

### DISCUSSION THREAD

**Discussion by yukitch** [Web3Auth/Web3Auth#1095] [Help]
How firebase authentication login users can login OpenSea? or any Dapps?
https://web3auth.io/docs/guides/firebase#authentication

I founded torus wallet connect but, a firebase authentication user cannot connect OpenSea with it. 
<img width="718" alt="image" src="https://user-images.githubusercontent.com/5962244/208234378-dafe3783-3ff0-4ca5-862d-3e095809de8d.png">

**Comment by shahbaz17** [comment]
Firebase authentication ( or a JWT-based login ) is not supported on Torus Wallet.
